### PR TITLE
[mage] FrostBomb, UnstableMagic, and ArcticGale

### PR DESCRIPTION
**Pull request by kfinch**
Added modules for Unstable Magic and Arctic Gale. I intended to also add the last Frost Mage lvl90 talent, but I literally cannot find a single log of anyone using Frost Bomb, and I don't want to ship a module I can't test.

### CODE REVIEW

File: src/Parser/Mage/Fire/CHANGELOG.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export default `
-17-10-2017 - Added Mirror Image Module. (by Sref)
+17-10-2017 - Added Unstable Magic and Arctic Gale modules. (by Sref)
```
Comment (sMteX) [inline src/Parser/Mage/Fire/CHANGELOG.js]: Should be only Unstable Magic in Fire.

Comment (kfinch) [reply src/Parser/Mage/Fire/CHANGELOG.js]: Copy-paste, the ancient enemy

**Comment by kfinch** [comment]
Fixed

**Comment by kfinch** [comment]
Added Frost Bomb as well, thanks Sharrq for the logs.

File: src/Parser/Mage/Frost/Modules/Features/FrostBomb.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  hits = 0;
+  procs = 0;
+  hitTimestamp;
+
+  on_initialized() {
+	   this.active = this.combatants.selected.hasTalent(SPELLS.FROST_BOMB_TALENT.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if(event.ability.guid === SPELLS.FROST_BOMB_DAMAGE.id) {
```
Comment (rp4rk) [inline src/Parser/Mage/Frost/Modules/Features/FrostBomb.js]: The nesting isn't awful here but if this were to get worse you could bomb out by checking for a non-equivalency and returning.

**Comment by kfinch** [comment]
@rp4rk which blocks are strange? Is it the proc counter stuff?

**Comment by rp4rk** [comment]
Mainly in regards to some of the nesting, there are some areas that could benefit from using functions for labelling your logic.

**Pull request merged**